source/codegen/templates/service_helpers.mako

```diff
@@ -164,12 +190,24 @@ ${initialize_standard_input_param(function_name, parameter)}\
       ${c_type} ${parameter_name} = ${request_snippet}.c_str();\
 % elif c_type == 'ViString' or c_type == 'ViRsrc':
       ${c_type} ${parameter_name} = (${c_type})${request_snippet}.c_str();\
-% elif c_type == 'ViInt8[]' or c_type == 'ViChar[]':
```

Comment:
are we no longer using the VIChar[]?

Reply:
Looks like both of these checks are moved to `is_string_arg` helper below.